google-cloud-logging/lib/google/cloud/logging/middleware.rb

```diff
@@ -74,6 +74,56 @@ def extract_trace_id env
         end
 
         ##
+        # Construct monitoring resource based on given type and label (both are
```

Comment:
"Construct a monitored resource based on the given type and label if both are provided. Otherwise, construct a default monitored resource based on the current environment."